### ISSUE TITLE
Preview images on messages index pages don't contain alt text

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#33]
# Describe the bug
The preview images for each clip on the drafts page and completed messages page do not contain alt text. This is a challenge for anyone using assistive technologies. Particularly because those images are linked to the specific clips they represent.

## Expected behavior
Each image should have the clip's subject (which is included in the image itself) included in the ALT text

## Screenshots
![image](https://user-images.githubusercontent.com/342510/107133484-3d7afa80-68ae-11eb-8970-69c5395fb3b4.png)

## Additional context
[Using alt attributes on img elements](https://www.w3.org/TR/WCAG20-TECHS/H37.html)

**Comment by robby1066** [comment]
Went through and ran the WAVE extension test on all pages and cleaned up any errors. Most were missing `alt` tags, but also got several other random problems fixed.

**Comment by robby1066** [comment]
Tested and deployed on Feb 8, 2021
